### PR TITLE
docs(upgrade): fix `upgrade` docs examples for v13

### PR DESCRIPTION
**Pull request by gkalpak**
_Builds on top of #43880._

##
This PR primarily fixes the SystemJS docs examples to be compatible with Angular v13 and re-enables the associated tests on CI.

See the individual commits for details.

### CODE REVIEW

File: aio/content/guide/upgrade-setup.md [hunk 1]
```diff
@@ -123,6 +92,125 @@ Open a terminal window in the project folder and enter the following commands fo
...
+        '@angular/compiler': 'npm:@angular/compiler/fesm2015/compiler.mjs',
+        '@angular/platform-browser': 'npm:@angular/platform-browser/fesm2015/platform-browser.mjs',
+        '@angular/platform-browser-dynamic': 'npm:@angular/platform-browser-dynamic/fesm2015/platform-browser-dynamic.mjs',
+        '@angular/router': 'npm:@angular/router/fesm2015/router.mjs',
+        '@angular/forms': 'npm:@angular/forms/fesm2015/forms.mjs',
+        // ...
+      },
+      // ...
+    });
+    ```
```
Comment (petebacondarwin) [inline aio/content/guide/upgrade-setup.md:134]: Could this not be externalized, given that we have a load of `systemjs.config.js` files in our examples?

Comment (petebacondarwin) [reply aio/content/guide/upgrade-setup.md:134]: And similarly for the snippets below...

**Review by petebacondarwin — APPROVED** [review]
One suggestion about the example snippets but happy to approve the rest.

File: aio/content/guide/upgrade-setup.md (hunk 1 above)
Comment (gkalpak) [reply aio/content/guide/upgrade-setup.md]: Good point! I forgot to do that. Fixed now.

**Comment by atscott** [comment]
This PR was merged into the repository by commit 37e02750a707eaaa859e87c0c55a39a1c5361ec1.

**Pull request closed without merging**